Examples/OTLP Exporter/main.swift

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Foundation
+import OpenTelemetryProtocolExporter
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import ResourceExtension
+import StdoutExporter
+import ZipkinExporter
+import SignPostIntegration
+import GRPC
+import NIO
+import NIOSSL
+
+let sampleKey = "sampleKey"
+let sampleValue = "sampleValue"
+
+var resources = DefaultResources().get()
+
+let instrumentationLibraryName = "OTLPExporter"
+let instrumentationLibraryVersion = "semver:0.1.0"
+var instrumentationLibraryInfo = InstrumentationLibraryInfo(name: instrumentationLibraryName, version: instrumentationLibraryVersion)
+
+var tracer: TracerSdk
+tracer = OpenTelemetrySDK.instance.tracerProvider.get(instrumentationName: instrumentationLibraryName, instrumentationVersion: instrumentationLibraryVersion) as! TracerSdk
+
+let configuration = ClientConnection.Configuration(
+    target: .hostAndPort("localhost", 4317),
+    eventLoopGroup: MultiThreadedEventLoopGroup(numberOfThreads: 1)
+)
+let client = ClientConnection(configuration: configuration)
+
+let otlpTraceExporter = OtlpTraceExporter(channel: client)
+let stdoutExporter = StdoutExporter()
+let spanExporter = MultiSpanExporter(spanExporters: [otlpTraceExporter, stdoutExporter])
+
+let spanProcessor = SimpleSpanProcessor(spanExporter: spanExporter)
+OpenTelemetrySDK.instance.tracerProvider.addSpanProcessor(spanProcessor)
+
+if #available(macOS 10.14, *) {
+    OpenTelemetrySDK.instance.tracerProvider.addSpanProcessor(SignPostIntegration())
+}
+
+func createSpan() {
+    let parentSpan = tracer.spanBuilder(spanName: "Main").setSpanKind(spanKind: .client).startSpan()
+    parentSpan.setAttribute(key: sampleKey, value: sampleValue)
+    for _ in 1...10 {
+        doWork(parentSpan: parentSpan)
+    }
+    Thread.sleep(forTimeInterval: 0.5)
+    parentSpan.end()
+}
+
+func doWork(parentSpan: Span) {
+    OpenTelemetry.instance.contextProvider.setActiveSpan(parentSpan)
```

Comment:
This line should moved to the `createSpan()` function after parent span has been created (can be called before or after the `setAttribute()` method), since the active span is stored between function calls. Calling it here repeatedly is superfluous. The `parentSpan` parameter can be omitted then